amazon/ion/simpleion.py

```diff
@@ -457,6 +457,9 @@ def loads(ion_str, catalog=None, single_value=True, encoding='utf-8', cls=None,
             ``sequence_as_stream=True``, it must be loaded using ``single_value=False``. Default: True.
         parse_eagerly: (Optional[True|False]) Used in conjunction with ``single_value=False`` to return the result as list
             or an iterator
+        text_buffer_size_limit: The size maximum size allowed for text, 512 bytes is the default. This option only works
+            when C extension is enabled (and it is enabled by default). Raising text buffer limit is required for text
```

Comment:
This could potentially give users the impression that when the C extension is disabled, it is not possible to raise the limit above 512 bytes. In actuality, when the C extension is disabled, there is no limit.

I recommend the following:

> The maximum byte size allowed for text values when the C extension is enabled (default: 512 bytes). This option only has an effect when the C extension is enabled (and it is enabled by default). When the C extension is disabled, there is no limit on the size of text values.

Reply:
Thanks, below commit addressed it.